tests/server.test.js

```diff
@@ -0,0 +1,52 @@
+const { MongoMemoryServer } = require('mongodb-memory-server');
+const supertest = require('supertest');
+const initByDbConn = require('../server/scripts/db.init_by_dbconn');
+const createApp = require('../server/app');
+
+let app;
+let db;
+
+describe('Test /api/ path', () => {
```

Comment:
@IgorChaikin Lets use `describe()` for describing testing route, and `it()` for specifying different test cases for this route:

```js
describe(`GET /api/admin/users`, () => {
  it('should load all users', async () => {
    const response = await agent.get(`/api/admin/users`).set('Cookie', cookie);

    expect(response.status).toBe(200);
    expect(response.body.results).toBeType('array');
    expect(response.body.results).toContainEqual(
      expect.objectContaining({
        id: expect.any(String),
        name: expect.any(String),
        email: expect.any(String),
        phone: expect.any(String),
        admin: expect.any(Boolean),
      }),
    );
  });
});

describe(`PUT /api/admin/users`, () => {
  it('should change auth password if password field is passed', async () => {
    ...
  });

  it("shouldn't change auth password if password field is not passed", async () => {
    ...
  });
})
```

Reply:
@IgorChaikin For api test, could you please modify structure, as I described above. Use `describe('[ROUTE]')` to combine test cases for a specific route, for example:

```js
describe(`POST /api/auth/login`, () => {
  it('...', () => {})
  it('...', () => {})
  it('...', () => {})
})
```